test/performance/README.md

```diff
@@ -1,128 +1,187 @@
-## Stress Test Experiments
+## Workload Benchmark for KubeStellar
 
+To generate the sample workload for KubeStellar performance experiments proceed as following:
 
-## Generate the workload traffic
+### a) Generate the workload traffic
 
-1. Clone the following fork of the clusterloader2 repo: 
+1. Clone the following clusterloader2 repo: 
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
 
-2. Configure the parameters of your experiment:  
+   Set the variable `WORK_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) Plain K8s clusters:
+
+   ```bash
+   $ ./setup-clusterloader2.sh
+   ```
+
+   In Kubernetes environments, a modified version of the kube-burner [cluster-density](https://github.com/kube-burner/kube-burner/tree/main/examples/workloads/cluster-density) workload will be generated per namespace. This workload consists of the following objects:
```

Comment:
I would emphasize "in _plain_ Kubernetes environments, ..."

Reply:
Fixed!